.github/workflows/ci-schedule.yml

```diff
@@ -0,0 +1,39 @@
+name: CI Workflow
+on:
+  schedule:
+    # Run this workflow "At 18:00 UTC on Sunday and Saturday"
+    - cron: '0 18 * * 0,6'
+jobs:
+  e2e:
+    name: e2e test
+    # prevent job running from forked repository
+    if: ${{ github.repository == 'karmada-io/karmada' }}
+    runs-on: ubuntu-20.04
```

Comment:
Do we need to configure the branch, i.e., only the master branch?

Reply:
```
In GitHub, the base branch is the branch that you want to merge changes into. When you create a pull request, you specify the base branch and the branch that contains the changes you want to merge. The base branch is typically the "main" or "master" branch of the repository, but it can be any existing branch in the repository.

The default branch is the branch that appears as the initial branch when someone visits the repository on GitHub. This is the branch that is checked out when someone clones the repository or visits it on the web. When you create a new repository, GitHub sets the default branch to "main", unless configured otherwise. You can change the default branch in your repository settings.
```

So, the release branch is a base branch, because we need to merge some commits into it during releasing period.

Reply:
For now, we only need to add the schedule workflow to `master`, the workflow will be run on/against master.
After we create a new branch like `release-1.6`, the workflow will be run on/against both `master` and `release-1.6`.
Is it right?

Reply:
In fact, the `schedule` event does not support setting branches. We can modify a specified branch like this way: https://stackoverflow.com/questions/58798886/github-actions-schedule-operation-on-branch

Scheduled workflows run on the latest commit on the default or base branch. In my personal repository settings, I set master as the default branch: https://github.com/XiShanYongYe-Chang/karmada/settings/branches

The final run effect also runs only on the master branch.

![image](https://user-images.githubusercontent.com/78244870/223316523-e5b541ee-4835-4344-8f33-0fc8b101b2c4.png)